### PR TITLE
New nav dropdown

### PR DESCRIPTION
**Pull request by dospore**
### Motivation

- old nav dropdown just wasnt very responsive / tidy

### Changes

- seperate out nav dropdown into components
- add new dropdown

### CODE REVIEW

**Comment by dospore** [comment]
- might need to pull this one down and check all the styles

**Review by Steven-Yulong-Yan — DISMISSED** [review]
This is of the highest quality. I wish all of tracer-website were like this

File: package.json [hunk 1]
```diff
@@ -64,6 +64,7 @@
         "@storybook/addon-essentials": "^6.1.21",
         "@storybook/addon-links": "^6.1.21",
         "@storybook/react": "^6.1.21",
+        "@svgr/webpack": "^5.5.0",
```
Comment (Steven-Yulong-Yan) [inline package.json:67]: What is this for?

File: components/Nav/Navbar/MobileSocialLogos.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Icon, { GithubOutlined, TwitterOutlined } from '@ant-design/icons';
+import styled from 'styled-components';
+//@ts-ignore
+import DiscordLogo from 'public/img/logos/socials/discord-logo-white.svg';
+//@ts-ignore
+import DiscourseLogo from 'public/img/logos/socials/discourse-logo-white.svg';
```
Comment (Steven-Yulong-Yan) [inline components/Nav/Navbar/MobileSocialLogos.tsx:7]: Differently sourced images

**Review by Steven-Yulong-Yan — APPROVED** [review]

File: components/Nav/Navbar/MobileSocialLogos.tsx (hunk 2 above)
Comment (dospore) [reply components/Nav/Navbar/MobileSocialLogos.tsx:7]: yeah its just how to work with antd/Icon so all the styles get applied, alternatively we could get all the socials icons https://ant.design/components/icon/

**Pull request merged**